### ISSUE TITLE
Add assembly stats summary to MultiQC report

### ISSUE THREAD

**Issue by b16joski** [ctmrbio/BACTpipe#43]
To gather assembly stats.

**Comment by abhi18av** [comment]
@thorellk hasn't this been included in the `3.0` release?

**Comment by thorellk** [comment]
Not really. The assembly summary stats file is output after the Shovill process but MultiQC does not yet have support for statswrapper.sh outputs. So either we have to add this functionality to MultiQC (it already has support for several other bbmap tools but not statswrapper) or change the tool used for assembly statistics. This would be an enhancement but is in no way necessary.
